lib/tapioca/generator.rb

```diff
@@ -307,6 +308,32 @@ def constantize(constant_names)
       constant_map.values
     end
 
+    sig { params(generator_names: T::Array[String]).returns(T::Array[T.class_of(Compilers::Dsl::Base)]) }
+    def constantize_generators(generator_names)
+      generator_map = generator_names.map do |name|
+        # Try to find built-in tapioca generator first, then globally defined generator. The
+        # explicit `break` ensures the class is returned, not the `potential_name`.
+        generator_klass = ["Tapioca::Compilers::Dsl::#{name}", name].find do |potential_name|
+          break Object.const_get(potential_name)
+        rescue NameError
+          # Skip if we can't find generator by the potential name
+        end
+
+        [name, generator_klass]
+      end.to_h
+
+      unprocessable_generators = generator_map.select { |_, v| v.nil? }
```

Comment:
It would have to be an `if` after that internal `find` since we run through that rescue multiple times and I think that would probably be less readable since you'd have to explicitly `return nil` in the map and then `compact` after.

A more readable middle-ground might be an `each_with_object` or just initializing those enumerables outside of the loop:
```ruby
generator_map = {}
unprocessable_generators = []

generator_names.each do |name|
  # Try to find built-in tapioca generator first, then globally defined generator.
  ["Tapioca::Compilers::Dsl::#{name}", name].each do |potential_name|
    generator_map[name] = Object.const_get(potential_name)
    break
  rescue NameError
    # Skip if we can't find generator by the potential name
  end

  unprocessable_generators << name unless generator_map[name]
end
```

I could also update the `constantize` method right above this to follow a similar pattern since that's what I was following.